builder/builder-next/controller.go

```diff
@@ -121,9 +123,13 @@ func newSnapshotterController(rt http.RoundTripper, opt Opt) (*mobycontrol.Contr
 		ResolveCacheImporterFuncs: map[string]remotecache.ResolveCacheImporterFunc{
 			"registry": localinlinecache.ResolveCacheImporterFunc(opt.SessionManager, opt.RegistryHosts, wa.ContentStore(), dist.ReferenceStore, dist.ImageStore),
 			"local":    localremotecache.ResolveCacheImporterFunc(opt.SessionManager),
+			"gha":      gha.ResolveCacheImporterFunc(),
 		},
 		ResolveCacheExporterFuncs: map[string]remotecache.ResolveCacheExporterFunc{
-			"inline": inlineremotecache.ResolveCacheExporterFunc(),
+			"registry": registryremotecache.ResolveCacheExporterFunc(opt.SessionManager, opt.RegistryHosts),
+			"inline":   inlineremotecache.ResolveCacheExporterFunc(),
+			"local":    localremotecache.ResolveCacheExporterFunc(opt.SessionManager),
+			"gha":      gha.ResolveCacheExporterFunc(),
```

Comment:
For a follow-up, as these are basically defined by BuildKit (?) we should look if there's consts or code that we can share from BuildKit, so that there's a canonical source of truth for acceptable values.